tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc

```diff
@@ -145,13 +146,15 @@ llvm::Optional<Value> convertPackOp(PatternRewriter& rewriter, Operation* op,
   RankedTensorType input_type =
       op->getOperand(0).getType().dyn_cast<RankedTensorType>();
   if (!input_type) {
-    op->emitOpError("PackOp: input type not ranked tensor");
+    (void)rewriter.notifyMatchFailure(op,
+        "PackOp: input type not ranked tensor");
     return llvm::None;
   }
 
   input_type = inputs[0].getType().dyn_cast<RankedTensorType>();
   if (!input_type) {
-    op->emitOpError("Input 0 type not ranked tensor.");
+    (void)rewriter.notifyMatchFailure(op,
+        "Input 0 type not ranked tensor.");
```

Comment:
input for consistency with others (sentence fragment, starting lower case no trailing punctuation)